main.js

```diff
@@ -101,9 +102,31 @@ function createInstance(activeClass) {
 }
 
 function startActivity() {
-  createCurrentActivity();
-  // document.getElementById('user-accomplish').innerText = currentActivity.description;
-  document.getElementById('user-accomplish').innerText = currentActivity.description;
-  document.getElementById('user-minutes').innerText = currentActivity.minutes;
-  document.getElementById('user-seconds').innerText = `:${currentActivity.seconds}`;
+  var inputs = [accomplishInput, minutesInput, secondsInput];
+  isCatChosen();
+  areInputsDefined(inputs);
+  if (errorMsg[0].classList.contains('hidden') && errorMsg[1].classList.contains('hidden') && errorMsg[2].classList.contains('hidden') && errorMsg[3].classList.contains('hidden')) {
+    createCurrentActivity();
+    document.getElementById('user-accomplish').innerText = currentActivity.description;
+    document.getElementById('user-minutes').innerText = currentActivity.minutes;
+    document.getElementById('user-seconds').innerText = `:${currentActivity.seconds}`;
+  }
+}
+
+function isCatChosen() {
+  if (!studyButton.classList.contains('study-active') && !meditateButton.classList.contains('meditate-active') && !exerciseButton.classList.contains('exercise-active')) {
+    errorMsg[0].classList.remove('hidden');
+  } else {
+    errorMsg[0].classList.add('hidden');
+  }
+}
+
+function areInputsDefined(userInputs) {
+  for (var i = 0; i < userInputs.length; i++) {
+    if (userInputs[i].value === '') {
+      errorMsg[i + 1].classList.remove('hidden');
```

Comment:
good reminder of "i+1", reminder for the first index of error array being with the categories